src/features/marketplace/components/Collection.tsx

```diff
@@ -4,7 +4,7 @@ import { loadMarketplace as loadMarketplace } from "../actions/loadMarketplace";
 import * as Auth from "features/auth/lib/Provider";
 import { useActor } from "@xstate/react";
 import { useNavigate, useSearchParams } from "react-router-dom";
-import { ListViewCard } from "./ListViewCard";
+import { useLocation, ListViewCard } from "./ListViewCard";
```

Comment:
This is breaking the app. Small typo,`useLocation` should be imported from just above in `react-router-dom`

Reply:
Classic